### PR TITLE
Adds new colours of striped scarves.

### PR DESCRIPTION
**Pull request by nesisknesis**
Creates the stripedscarf subtype instead of them being individual defines, allowing for less effort to colour *each* sprite. 

Colours include all the non-striped ones except white, and are available in the Clothesmate.

#### Changelog

:cl:  Identification
rscadd: New colours of striped scarves are available in the clothesmate. 
tweak: stripedscarf subtype replaces striped[colour]scarf defines
/:cl:

### CODE REVIEW

**Review by fluffe9911 — CHANGES_REQUESTED** [review]
Nice scarfs

File: code/modules/clothing/neck/_neck.dm [hunk 1]
```diff
@@ -170,26 +170,76 @@
 	icon_state = "christmasscarf"
 	item_color = "christmasscarf"
 
-//The three following scarves don't have the scarf subtype
+//The three following scarves don't have the scarf subtype // yogs - they now have the striped scarf subtype
 //This is because Ian can equip anything from that subtype
 //However, these 3 don't have corgi versions of their sprites
-/obj/item/clothing/neck/stripedredscarf
+
+/obj/item/clothing/neck/stripedredscarf //yogs - needs to exist because its mapped in :annoyed:
```
Comment (Altoids1) [inline code/modules/clothing/neck/_neck.dm]: You can't remove all these scarftypes and replace them with your yogs variant. That's just more trouble for me down the road.

Dump all your new scarftypes into ``/yogstation/code/modules/clothing/neck/_neck.dm``.

**Review by Altoids1 — CHANGES_REQUESTED** [review]

File: code/modules/vending/clothesmate.dm [hunk 2]
```diff
@@ -33,9 +33,15 @@
 		            /obj/item/clothing/neck/scarf/black = 3,
 		            /obj/item/clothing/neck/scarf/zebra = 3,
 		            /obj/item/clothing/neck/scarf/christmas = 3,
-		            /obj/item/clothing/neck/stripedredscarf = 3,
-		            /obj/item/clothing/neck/stripedbluescarf = 3,
-		            /obj/item/clothing/neck/stripedgreenscarf = 3,
+		            /obj/item/clothing/neck/yogs/stripedscarf/black = 3, //yogs start
```
Comment (fluffe9911) [inline code/modules/vending/clothesmate.dm]: Move these scarfs at the end of the clothes mate file where all the other yogs stuff is for max modularization

Comment (nesisknesis) [reply code/modules/vending/clothesmate.dm]: donso

**Review by Altoids1 — APPROVED** [review]

**Pull request merged**